app/models/incident.rb

```diff
@@ -104,43 +104,53 @@ def csv_row
     row
   end
 
-  def export_claims_xml!
+  def export_claims_xml(current_user)
     report = driver_incident_report
     builder = Nokogiri::XML::Builder.new do |doc|
       doc.claims_incident_data do
-        doc.incident do
-          doc.date_entered        Date.today.strftime('%Y-%m-%d')
```

Comment:
Is `doc.date_entered` not important?

Reply:
This field was not specified in the list that Liza gave us, but I will check with her again.